homeassistant/components/velbus/__init__.py

```diff
@@ -34,6 +36,16 @@
 
 CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
 
+type VelbusConfigEntry = ConfigEntry[VelbusData]
+
+
+@dataclass
+class VelbusData:
+    """Runtime data for the Velbus config entry."""
+
+    cntrl: Velbus
```

Comment:
It would be nice to have vowels here as well

Reply:
its not increasing the readability, but i just did it